app/api/resources/organizations.py

```diff
@@ -27,19 +27,19 @@
 
 
 @organizations_ns.response(
-        HTTPStatus.UNAUTHORIZED,
+        HTTPStatus.UNAUTHORIZED.value,
         f"{messages.TOKEN_HAS_EXPIRED}\n"
         f"{messages.TOKEN_IS_INVALID}\n"
         f"{messages.AUTHORISATION_TOKEN_IS_MISSING}"
 )
-@organizations_ns.response(HTTPStatus.FORBIDDEN, f"{messages.NOT_ORGANIZATION_REPRESENTATIVE}")
-@organizations_ns.response(HTTPStatus.INTERNAL_SERVER_ERROR, f"{messages.INTERNAL_SERVER_ERROR}")
+@organizations_ns.response(HTTPStatus.FORBIDDEN.value, f"{messages.NOT_ORGANIZATION_REPRESENTATIVE}")
+@organizations_ns.response(HTTPStatus.INTERNAL_SERVER_ERROR.value, f"{messages.INTERNAL_SERVER_ERROR}")
 @organizations_ns.route("organization")
 class Organization(Resource):
     @classmethod
     @organizations_ns.doc("get_organization")
-    @organizations_ns.response(HTTPStatus.OK, "Successful request", get_organization_response_model)
-    @organizations_ns.response(HTTPStatus.NOT_FOUND, f"{messages.ORGANIZATION_DOES_NOT_EXIST}")
+    @organizations_ns.response(HTTPStatus.OK.value, "Successful request", get_organization_response_model)
+    @organizations_ns.response(HTTPStatus.NOT_FOUND.value, f"{messages.ORGANIZATION_DOES_NOT_EXIST}")
```

Comment:
Goodjob 👏🏼. Looks good. This file is all you need. You can remove the other files from the commit.